### PR TITLE
Adding Node tutorial

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR brings a new "getting started" guide for the Node Chainguard Image, based on the PHP tutorial that worked as POC.

It also brings a sort of tentative implementation for reusable blurbs of text, I used templates to implement this so it's not super dynamic but works well for this use case. Maybe we find a better way in the future.

Preview: https://deploy-preview-349--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/reference/node/getting-started-node/

This PR closes https://github.com/chainguard-dev/internal/issues/1071

### CODE REVIEW

File: content/chainguard/chainguard-images/reference/node/getting-started-node.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+draft: false
+images: []
+menu:
+  docs:
+    parent: "node-guides"
+weight: 610
+toc: true
+---
+
+The Node Chainguard Image is a distroless container image that has the tooling necessary to build and execute Node applications, including  `npm`.
```
Comment (jamonation) [inline content/chainguard/chainguard-images/reference/node/getting-started-node.md:16]: Would a link to the reference page make sense here? I don't think we link right to github docs do we?

File: content/chainguard/chainguard-images/reference/node/getting-started-node.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+In this guide, we'll set up a demo application and create a Dockerfile to build and execute the demo using the Node Chainguard Image as base.
+
+This tutorial requires Docker, Node, and Npm to be installed on your local machine.
+
+{{< details "What is distroless" >}}
+{{< blurb/distroless >}}
```
Comment (jamonation) [inline content/chainguard/chainguard-images/reference/node/getting-started-node.md:23]: 🤯

File: content/chainguard/chainguard-images/reference/node/getting-started-node.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+{{< blurb/wolfi >}}
+{{< /details >}}
+
+{{< details "Chainguard Images" >}}
+{{< blurb/images >}}
+{{< /details >}}
+
+## Step 1: Setting up a Demo Application
+
+We'll start by creating a basic Node application to serve as a demo. This application uses a mock server to test GET and POST requests, and is based on the [Docker Node example](https://docs.docker.com/language/nodejs/build-images/).
```
Comment (jamonation) [inline content/chainguard/chainguard-images/reference/node/getting-started-node.md:36]:
```suggestion
We'll start by creating a small Node application to serve as a demo. This application uses a mock server to test GET and POST requests, and is based on the [Docker Node example](https://docs.docker.com/language/nodejs/build-images/).
```

File: content/chainguard/chainguard-images/reference/node/getting-started-node.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+curl http://localhost:8000/test
+```
+
+You'll get output similar to this:
+
+```shell
+{"code":"success","meta":{"total":1,"count":1},"payload":[{"msg":"testing","id":"f427f835-3e93-43ad-91c8-d150dffba0f9","createDate":"2023-02-07T14:48:54.256Z"}]}
+```
+
+The demo application is now ready. In the next step, you'll create a Dockerfile to run your app.
```
Comment (jamonation) [inline content/chainguard/chainguard-images/reference/node/getting-started-node.md:108]: I went ahead with the next section and got this message:

```
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:8000 -> 0.0.0.0:0: listen tcp 0.0.0.0:8000: bind: address already in use.
ERRO[0000] error waiting for container: context canceled
```

So I think a sentence or two here at the end to remind the user to CTRL/CMD+C their running `node server.js` would be helpful.

File: layouts/shortcodes/blurb/wolfi.html [hunk 5]
```diff
@@ -0,0 +1 @@
+Wolfi is a community Linux undistro created specifically for containers. This brings distroless to a new level, including additional features targeted at securing the software supply chain of your application environment: comprehensive SBOMs, signatures, daily updates, and timely CVE fixes.
```
Comment (jamonation) [inline layouts/shortcodes/blurb/wolfi.html:1]: Worth a link to the [Github org](https://github.com/wolfi-dev/)? I dunno, just a thought!

**Review by jamonation — APPROVED** [review]
Looks great, works nicely, and I am already thinking about how we can start using the shortcode snippets for other things around the Academy.

File: content/chainguard/chainguard-images/reference/node/getting-started-node.md (hunk 1 above)
Comment (erikaheidi) [reply content/chainguard/chainguard-images/reference/node/getting-started-node.md:16]: good catch!

**Pull request merged**